tests/test_advisory.py

```diff
@@ -70,5 +71,8 @@ def test_get_security_alerts(self):
             self.assertFalse(ad.has_blocking_secruity_alert(), f"advisory {ad.errata_id} has blocking security alerts")
 
     def test_kernel_tag(self):
-       self.assertTrue(Advisory(errata_id=144853, impetus='image').check_kernel_tag())
-       self.assertFalse(Advisory(errata_id=144854, impetus='metadata').check_kernel_tag())
+        ads = self.am.get_advisories()
```

Comment:
I think we don't need this code block, just verify the return value of check_kernel_tag is good enough.